plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -24,6 +24,7 @@ go_library(
         "//plugin/pkg/scheduler/schedulercache:go_default_library",
         "//vendor:k8s.io/apimachinery/pkg/apis/meta/v1",
         "//vendor:k8s.io/apimachinery/pkg/labels",
+        "//vendor:k8s.io/client-go/pkg/apis/apps",
```

Comment:
Oh, sorry I made a wrong rebase and had to resubmit the changes. I wanted to respond individually to your points.
I initially wanted to use k8s.io/kubernetes/pkg/apis/apps/v1beta1 instead of apps too, but StatefulSet does not exist in k8s.io/kubernetes/pkg/apis/apps/v1beta1.